docs/design/admission_control_resource_quota.md

```diff
@@ -54,109 +58,139 @@ const (
+volatile.
 
-The quota is only enforced if the kube-apiserver is started as follows:
+As a consequence of this decision, the user is able to spread its usage of a resource across multiple tiers
+of service.  Let's demonstrate this via an example with a 4 cpu quota.
 
-```console
-$ kube-apiserver -admission_control=ResourceQuota
-```
+The quota may be allocated as follows:
+
+| Pod | Container | Request CPU | Limit CPU | Tier | Quota Usage |
+| --- | --------- | ----------- | --------- | ---- | ----------- |
+| X | C1 | 1 | 4 | Burstable | 1 |
+| Y | C2 | 2 | 2 | Guaranteed | 2 |
+| Z | C3 | 1 | 3 | Burstable | 1 |
 
-## kube-controller-manager
+It is possible that the pods may consume 9 cpu over a given time period depending on the nodes available cpu
+that held pod X and Z, but since we scheduled X and Z relative to the request, we only track the requesting
+value against their allocated quota.  If one wants to restrict the ratio between the request and limit,
+it is encouraged that the user define a **LimitRange** with **LimitRequestRatio** to control burst out behavior.
+This would in effect, let an administrator keep the difference between request and limit more in line with
+tracked usage if desired.
 
-A new controller is defined that runs a synch loop to calculate quota usage across the namespace.
+## Status API
 
-**ResourceQuota** usage is only calculated if a namespace has a **ResourceQuota** object.
+A REST API endpoint to update the status section of the **ResourceQuota** is exposed.  It requires an atomic compare-and-swap
+in order to keep resource usage tracking consistent.
 
-If the observed usage is different than the recorded usage, the controller sends a **ResourceQuotaUsage** resource
-to the server to atomically update.
+## Resource Quota Controller
 
-The synchronization loop frequency will control how quickly DELETE actions are recorded in the system and usage is ticked down.
+A resource quota controller monitors observed usage for tracked resources in the **Namespace**.
+
+If there is observed difference between the current usage stats versus the current **ResourceQuota.Status**, the controller
+posts an update of the currently observed usage metrics to the **ResourceQuota** via the /status endpoint.
+
+The resource quota controller is the only component capable of monitoring and recording usage updates after a DELETE operation
+since admission control is incapable of guaranteeing a DELETE request actually succeeded.
+
+## AdmissionControl plugin: ResourceQuota
+
+The **ResourceQuota** plug-in introspects all incoming admission requests.
+
+To enable the plug-in and support for ResourceQuota, the kube-apiserver must be configured as follows:
+
+```
+$ kube-apiserver -admission_control=ResourceQuota
+```
+
+It makes decisions by evaluating the incoming object against all defined **ResourceQuota.Status.Hard** resource limits in the request
```

Comment:
I will update the sentence to just say "resource constraints" to avoid confusion.